src/components/20-molecules/cookie-disclaimer/story.js

```diff
@@ -1,31 +1,54 @@
 /* global document */
 import { storiesOf } from '@storybook/html';
+import { text, withKnobs } from '@storybook/addon-knobs';
+import { html, render } from 'lit-html';
 import './index';
 import Readme from './README.md';
 import withNoBorder from '../../../../.storybook/addons/no-border';
 
 storiesOf('Molecules/Cookie disclaimer', module)
   .addDecorator(withNoBorder)
+  .addDecorator(withKnobs)
   .addParameters({
     readme: {
       sidebar: Readme,
     },
   })
-  .add(
-    'Cookie disclaimer - Default',
-    () => `
-        <axa-cookie-disclaimer buttonname="Akzeptieren" title="This website uses cookies">
-        <p>Any Description for the cookie disclaimer</p>
-        <axa-link variant="arrowright-animated-white" href="https://axa.ch/de/informationen/datenschutz.html">
-          Data protection
-        </axa-link>
-        </axa-cookie-disclaimer>
+  .add('Cookie disclaimer - Default', () => {
+    const buttonname = text('buttonname', 'Akzeptieren');
+    const title = text('text', 'This website uses cookies');
+    const description = text(
+      'description (this is not a direct attribute of cookie disclaimer)',
```

Comment:
I would remove that explanation in the brackets everywhere.

Reply:
I would probably also write `Description` with uppercase since it is not an attribute. But not sure if that is good, up to you.

Reply:
This with uppercase makes sense to me. And I shortened the text in brackets.